src/main/java/core/basesyntax/Car.java

```diff
@@ -75,12 +77,10 @@ public int hashCode() {
 
     @Override
     public String toString() {
-        return "Car{" +
-                "model='" + model + '\'' +
-                ", engine=" + engine +
-                ", color='" + color + '\'' +
-                ", autoType=" + autoType +
-                ", weight=" + weight +
-                '}';
+        return "Car{" + "model='" + model
+                + '\'' + ", engine=" + engine
+                + ", color='" + color
+                + '\'' + ", autoType=" + autoType
+                + ", weight=" + weight + '}';
```

Comment:
previous formatting is better IMO

Reply:
+ should be on a new line due to checkstyle